Source/ARTLocalDevice.h

```diff
@@ -17,6 +17,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nullable, nonatomic, readonly) ARTDeviceIdentityTokenDetails *identityTokenDetails;
 
+/**
+ Device secret generated using random data with sufficient entropy. It's a sha256 digest encoded with base64.
+ */
+@property (nullable, nonatomic) ARTDeviceSecret *secret;
```

Comment:
are there any tests for `ARTDeviceSecret` in `ARTLocalDevice`? How do you know that everything works properly after this change?

Reply:
I just noticed the inconsistency between libraries and made a change. I'm not sure about the tests, as I don't maintain this library or even read objective-C that well 😅 . Can I hand over this PR to you?